cura/Settings/IntentManager.py

```diff
@@ -75,11 +76,12 @@ def getCurrentAvailableIntents(self) -> List[Tuple[str, str]]:
             # TODO: We now do this (return a default) if the global stack is missing, but not in the code below,
             #       even though there should always be defaults. The problem then is what to do with the quality_types.
             #       Currently _also_ inconsistent with 'currentAvailableIntentCategories', which _does_ return default.
-        quality_groups = application.getQualityManager().getQualityGroups(global_stack)
+        quality_groups = ContainerTree.getInstance().getCurrentQualityGroups()
+        # TODO: These quality nodes in that tree already contain the intent nodes. We can optimise this.
```

Comment:
A `TODO` here. You can remove it if it's not applicable any more.

Reply:
Fixed here: fc679758ef22031ab532a6461aaf9984d812dd1a